build.sbt

```diff
@@ -843,6 +843,12 @@ lazy val testUtils = (project in utils("test-utils")).
     }
   )
 
+lazy val jsonUtils = (project in utils("json-utils")).
```

Comment:
can we have everit-json-schema dependency here, instead of components-utils?

Reply:
We even should, thanks :)